packages/ui/client/components/views/ViewModuleGraph.vue

```diff
@@ -9,14 +10,31 @@ const props = defineProps<{
 
 const { graph } = toRefs(props)
 
+const fileDetailsSize = injectFileDetailsSize()
+
+const style = computed(() => {
+  // eslint-disable-next-line no-console
+  const size = fileDetailsSize.value
+  // TODO: we need to listen to size change, but only change the width
+  // TODO: I dont know why resize not working, only first time

```

Comment:
The graph currently does not recenter on resize.
This was necessary to have stable node positions when the view mode changes.

Otherwise, the graph would move a bit every time the view mode changes.
If this is the preferred behavior, just set `alphas.resize` in `module-graph.ts` to a small value larger than 0 (maybe 0.01).

This manual sizing approach (including the style) is not required.

Reply:
ok, the problem is that we can resize the split panes, and this PR tries to fix this, I'll check it.